### ISSUE TITLE
Epic: Update submission for correctness, completeness

### ISSUE THREAD

**Issue by jadudm** [GSA-TTS/FAC#1404]
## Description

[//]: # ( A plain language description of the epic. Who, what, why. )

**Who**: Auditors, auditees
**What**: Are columns in our workbooks in the exactly same order as the Census workbooks?
**Why**: Because people are *definitely* copy-pasting their data.

This is a manual walk of every workbook after it has been generated via our `make` process. It should be carried out by at least two people, working collaboratively; an individual can do this, but it requires careful attention to detail. 

Do several things for each workbook:

1. Verbally confirm the order of columns in our workbook against those from Census.
2. Attempt to copy-paste content from a Census workbook to ours. It should align and paste in without error.

```[tasklist]
### Stories
- [ ] https://github.com/GSA-TTS/FAC/issues/1727
- [ ] https://github.com/GSA-TTS/FAC/issues/1728
- [ ] https://github.com/GSA-TTS/FAC/issues/1729
- [ ] https://github.com/GSA-TTS/FAC/issues/1920
- [ ] https://github.com/GSA-TTS/FAC/issues/1730
```

**Comment by Carley-Sullivan** [comment]
@LeoFotovich Here is a link to the Census site: https://facides.census.gov/Account/Login.aspx  and here is a link to the GSA FAc dev environment: https://fac-dev.app.cloud.gov/ Please create an account if you have not already done so.

**Comment by mogul** [comment]
@jadudm Do you want to convert these to issues so we can start tracking them on the story board give them MoSCoW priority, see them in the Epic view there, etc?

**Comment by LeoFotovich** [comment]
@jadudm  @mogul Apologies for the wait on this one! I was waiting for @Carley-Sullivan to look over this before submitting what I've found and put as notes. She hasn't looked at it yet but I can just leave it here in case anyone is curious.

Here is what I have so far. It stops at auditee certification due to the GSA FAC being buggy upon reaching that step.

https://docs.google.com/presentation/d/15uUerUBLHY_hRT7dGXGIOdEKbnZTVEE-zLMaI7N9dT0/edit?usp=sharing
